gui/src/components/commons/BaseModal.tsx

```diff
@@ -17,14 +17,17 @@ export function BaseModal({
       overlayClassName={
         props.overlayClassName ||
         classNames(
-          'fixed top-0 right-0 left-0 bottom-0 flex flex-col justify-center items-center w-full h-full bg-background-90 bg-opacity-60 z-20'
+          'fixed top-0 right-0 left-0 bottom-0 flex flex-col justify-center',
```

Comment:
Please use multiline strings (with backticks), classnames has to run which might result in flicker.

Reply:
I find this kind of optimization unnecesary, mainly to put in this PR. Check because there is a lot of components that do this, make an issue for this :angry:

Reply:
Might as well just start right now here in the components you touch?

Reply:
i do not agree with this. Keep classNames please

Reply:
``classnames has to run which might result in flicker`` hello no wtf

Reply:
pff, fineeeeee